migration-roadmap/asset.md

```diff
@@ -0,0 +1,200 @@
+
+```html
+<div class="spectrum-Asset" id="asset-123">
+    <svg viewBox="0 0 128 128" class="spectrum-Asset-file" width="10">
+        <path
+            class="spectrum-Asset-fileBackground"
+            d="M24,126c-5.5,0-10-4.5-10-10V12c0-5.5,4.5-10,10-10h61.5c2.1,0,4.1,0.8,5.6,2.3l20.5,20.4c1.5,1.5,2.4,3.5,2.4,5.7V116c0,5.5-4.5,10-10,10H24z"
+        ></path>
+        <path
+            class="spectrum-Asset-fileOutline"
+            d="M113.1,23.3L92.6,2.9C90.7,1,88.2,0,85.5,0H24c-6.6,0-12,5.4-12,12v104c0,6.6,5.4,12,12,12h80c6.6,0,12-5.4,12-12V30.4C116,27.8,114.9,25.2,113.1,23.3z M90,6l20.1,20H92c-1.1,0-2-0.9-2-2V6z M112,116c0,4.4-3.6,8-8,8H24c-4.4,0-8-3.6-8-8V12c0-4.4,3.6-8,8-8h61.5c0.2,0,0.3,0,0.5,0v20c0,3.3,2.7,6,6,6h20c0,0.1,0,0.3,0,0.4V116z"
+        ></path>
+    </svg>
+</div>
+```
+
+</details>
+
+<details>
+<summary>Diff: Legacy (CSS main) → Spectrum 2 (CSS spectrum-two)</summary>
+
+**No differences found between main and spectrum-two branches.**
+
+</details>
+
+### CSS => SWC mapping
+
+#### Non-variant selectors
+
+| CSS selector             | Attribute or slot  | Status           |
+| ------------------------ | ------------------ | ---------------- |
+| `.spectrum-Asset`        | Base component     | Implemented      |
+| `.spectrum-Asset-file`   | File variant SVG   | Implemented      |
+| `.spectrum-Asset-folder` | Folder variant SVG | Implemented      |
+| `.spectrum-Asset-image`  | Image content      | Missing from WC  |
+|                          | `label`            | Missing from CSS |
+|                          | Default slot       | Missing from CSS |
+
+#### Variant-specific selectors
+
+| CSS selector                       | Attribute or slot      | Status      |
+| ---------------------------------- | ---------------------- | ----------- |
+| `.spectrum-Asset-fileBackground`   | File background path   | Implemented |
+| `.spectrum-Asset-fileOutline`      | File outline path      | Implemented |
+| `.spectrum-Asset-folderBackground` | Folder background path | Implemented |
+| `.spectrum-Asset-folderOutline`    | Folder outline path    | Implemented |
+
+## Summary of changes
+
+### CSS => SWC changes
+
+The asset component has a relatively straightforward mapping between CSS and web components. There do not appear to be any significant differences from CSS that we might consider bringing into 2nd gen web components.
+
+**Implementation differences:**
+
+- **Variant system**: Web component uses a `variant` attribute (file, folder) to determine which SVG to render, while CSS uses preset parameters in templates
+- **Image support**: CSS uses a dedicated `image` parameter that renders `<img class="spectrum-Asset-image">`, while web component uses the default slot for custom content including images
+
+### CSS DOM structure changes
+
+No differences found between the legacy (CSS main) and Spectrum 2 (CSS spectrum-two) branches, as no component migration has been done.
```

Comment:
Did we still want to link to the CSS storybook at all? Or no, since this component isn't really migrated over there?

Reply:
I think we could link the CSS Storybook, but I'd be more inclined to link the Foundations version since there are no changes.